### PR TITLE
Dockerize bancho-service & pull secrets from vault

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.10
```
Comment (cmyui) [inline Dockerfile]:
```suggestion
FROM python:3.9
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env make
+
+build:
+	docker build -t bancho-service:latest .
+
+run:
+	docker run -p ${APP_PORT}:${APP_PORT} -it bancho-service:latest
```
Comment (cmyui) [inline Makefile]:
```suggestion
	docker run --network=host -it bancho-service:latest
```

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+run-bg:
+	docker run -p ${APP_PORT}:${APP_PORT} -d bancho-service:latest
```
Comment (cmyui) [inline Makefile]:
```suggestion
	docker run --network=host -d bancho-service:latest
```

File: scripts/bootstrap.sh [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+cd /srv/root
+
+if [[ -z "$PULL_SECRETS_FROM_VAULT" ]]; then
+  pip install -i $PYPI_INDEX_URL akatsuki-cli
+  akatsuki vault get bancho-service production -o .env
```
Comment (cmyui) [inline scripts/bootstrap.sh]: move env to $APP_ENV

File: .env.example [hunk 5]
```diff
@@ -36,3 +36,6 @@ WEBHOOK_AC_GENERAL=
 WEBHOOK_AC_CONFIDENTIAL=
 
 AMPLITUDE_API_KEY=
+
+SERVICE_READINESS_TIMEOUT=60
+PULL_SECRETS_FROM_VAULT=1
```
Comment (cmyui) [inline .env.example]:
```suggestion
PULL_SECRETS_FROM_VAULT=
```

File: scripts/bootstrap.sh [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+set -eo pipefail
+
+cd /srv/root
+
+if [ -z "$APP_ENV" ]; then
+  echo "Please set APP_ENV"
+  exit 1
+fi
+
+if [[ -z "$PULL_SECRETS_FROM_VAULT" ]]; then
```
Comment (tsunyoku) [inline scripts/bootstrap.sh]:
```suggestion
if [[ $PULL_SECRETS_FROM_VAULT -eq 1 ]]; then
```

File: scripts/bootstrap.sh [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+  pip install -i $PYPI_INDEX_URL akatsuki-cli
+  akatsuki vault get bancho-service $APP_ENV -o .env
```
Comment (cmyui) [inline scripts/bootstrap.sh:13]: random idea - maybe `$ akatsuki vault source` can also exist for when you don't need to persist to disk

**Pull request merged**